bower.json

```diff
@@ -15,5 +15,8 @@
   "ignore": [
     "**/.*",
     "!dist/braintree-angular.js"
-  ]
+  ],
+  "dependencies": {
+    "braintree-web": "2.5.3"
+  }
```

Comment:
I could have sworn that this was causing issues for me, but I am having trouble breaking it at the moment :confused:. Maybe this was only an issue when my wiredep was trying to link index.html into my app in the first place...
I can remove this change from the pull request. Does that also mean that braintree-web should be moved to a dev dependency in package.json?